### *_Умова задачі_*

*Ймовірність того, що телевізор витримає гарантій-ний
термін роботи, дорівнює 0,9 для всіх телевізорів, які обслуговує
гарантійна майстерня. Оцінити ймовірність того, що число телевізорів,
які витримають гарантійний термін роботи, буде в межах [85;95].*


### *_Опис рішення_*

1. Розраховуємо математичне очікування, а саме загальну кількість телевізорів, що мають витримати грарантійний термін роботи, а також розраховуємо дисперсію і стандартне відхилення.
2. Визначаємо  ε, що характеризує вибірку, а саме визначає допустиму різницю між нашою змінною Х та математичним очікуванням.
3. За допомогою нерівності Чебишева розраховуємо ймовірність того, що число телевізорів, які витримають гарантійний термін роботи, буде в межах [85;95].

### *_Рішення_*

In [2]:
from math import sqrt

In [6]:
X_j = 95
X_i = 90
n = 100
p = 0.9
q = 0.1

In [7]:
M_X = n*p
D_X =n*p*(q)
σ_x = sqrt(D_X)
print('Математичне очікування', M_X)
print('Стандартне відхилення', σ_x)

Математичне очікування 90.0
Стандартне відхилення 3.0


In [9]:
ε = X_j - M_X

P(|x-M(X)|≤ ε) ≥ 1 - DX/ε**2 

In [10]:
P_=1-D_X/ε**2
P_

0.64

#### _Відповідь: Ймовірність того ,що  що число телевізорів, які витримають гарантійний термін роботи, буде в межах [85;95] становить 64%._